File: trunk/quests/templeveeshan/Carx-Vean.pl
# link lord kreizenn and guards
#

sub EVENT_AGGRO {
  my $guard1 = $entity_list->GetMobByNpcTypeID(124074); #kreizenn
  my $guard2 = $entity_list->GetMobByNpcTypeID(124093); #kedrak
  if ($guard1) {
    my $call_guard1 = $guard1->CastToNPC();
    $call_guard1->AddToHateList($client, 1);
  }
  if ($guard2) {
    my $call_guard2 = $guard2->CastToNPC();
    $call_guard2->AddToHateList($client, 1);
  }
}

# EOF zone: templeveeshan ID: 124094 NPC: Carx-Vean

File: trunk/quests/templeveeshan/Carx-Vean.lua
-- Carx`Vean (124094) NPCID: 124094
function event_combat(e)
	if (e.joined) then
		-- grab the entity list
		local entity_list = eq.get_entity_list();
		-- also aggro #Lord_Kreizenn (124074) and Kedrak (124093) if they are up
		local npc_table = {124074,124093};
		for k,v in pairs(npc_table) do
			local npc = entity_list:GetMobByNpcTypeID(v);
			if (npc.valid) then
				npc:AddToHateList(e.other,1);
			end
		end
	end
end